Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.ControlBarAbstract.cls

```diff
@@ -63,7 +63,7 @@ arrangementOf: aView put: anObject
 			self invalidateLayout]!
 
 canAcceptSubViews
-	"Answer true if the receiver can accept aView sub views"
+	"Answer true if the receiver can accept aView sub-views"
```

Comment:
I failed to notice the extraneous `aView` here before, even though I commented about the hyphen